demo/netlify.toml

```diff
@@ -1,11 +1,14 @@
 [build]
 command = "npm run build"
 publish = "public/"
-environment = { GATSBY_CLOUD_IMAGE_CDN = "true" }
+environment = { NETLIFY_IMAGE_CDN = "true" }
```

Comment:
I think we will need to set `GATSBY_CLOUD_IMAGE_CDN` still because that's what `gatsby-plugin-utils` check to know wether it should skip doing the work at build? 

So ultimately we will need both `GATSBY_CLOUD_IMAGE_CDN` set (for gatsby side) and `NETLIFY_IMAGE_CDN` for the build plugin?

We probably can set `GATSBY_CLOUD_IMAGE_CDN` on user behalf when `NETLIFY_IMAGE_CDN` is set (in `onPreBuild` hook of build plugin)?

Reply:
Ah yes that's a good point, I added `NETLIFY_IMAGE_CDN` to the updated plugin-utils, but older versions will only check for `GATSBY_CLOUD_IMAGE_CDN`. So yes it will need to be both.